### DISCUSSION TITLE
Who is using Ark Ui? 👀

### DISCUSSION THREAD

**Discussion by vitalijalbu** [chakra-ui/ark#2198] [Show and tell]
Hi guys, let's **share here** the ark ui **showcase**, websites, apps, tools, everything... 🚀

**Comment by cschroeter** [comment]
Things I build with Ark UI include:


https://ark-ui.com
https://park-ui.com

**Upcoming** 
https://chakra-ui.com will use Ark UI in v4+

**Reply by vitalijalbu** [reply]
wow... **chakra ui v4**??!! 👀  any release date??
